rules/scripts/gdb_test_coordinator.py

```diff
@@ -156,7 +156,10 @@ def main(rom_kind: str = typer.Option(...),
         "; ".join([
             "adapter speed 200",
             "transport select jtag",
-            "reset_config trst_and_srst",
+            # Do not use srst as this will reset the entire chip on the FPGA,
+            # including the JTAG tap. Also the real chip does not have a srst line
+            # so we do not want to FPGA tests to differ from the real ones.
+            "reset_config trst_only",
```

Comment:
I agree that `srst` should not be used because it's only available on the FPGA. Note that we don't have `trst` connected, although it can be emulated with suitable `tms` and `tck` driving for the purpose of resetting the TAP controller.

Reply:
Different question: Does OpenOCD / GDB need a way to do a system reset? If so, how will that happen after this change?

Reply:
The GDB tests do not need a system reset. The only tests that require a system resset use opentitanlib that correctly resets the chip using a gpio line (e.g. for LC transitions).